ios/Classes/PluginExtensions/SwiftOneginiPlugin+Register.swift

```diff
@@ -46,8 +46,8 @@ extension SwiftOneginiPlugin: OneginiPluginRegisterProtocol {
         if let _typeValue = typeValue, let value = WebSignInType.init(rawValue: _typeValue) {
             type = value
         }
-        
         OneginiModuleSwift.sharedInstance.handleRegisteredProcessUrl(url ?? "", webSignInType: type)
+        result(nil)
```

Comment:
why?

Reply:
This is needed to resovle the promise, without this, the function will just return and never actually resolve leaving the future waiting on the Dart side.